specs/cm/change-mgt-spec.html

```diff
@@ -932,149 +932,14 @@ <h3>Usage Identifiers</h3>
       <section id="versionCompability" class="appendix">
         <h2>Version Compatibility with 2.0 Specifications</h2>
 
-        <section id="deprecatedTerms">
-          <h3>Deprecated terms</h3>
-
-          <p>
-            A number of terms introduced in early development of the OSLC Change Management domain were deprecated in
-            the finalized [[OSLCCM20]] specification. These terms are summarized here in order to indicate they remain
-            deprecated.
-          </p>
-
-          <table border="1" cellspacing="0" cellpadding="5">
-            <thead>
-              <tr>
-                <th><strong>Prefixed Name</strong></th>
-                <th><strong>Occurs</strong></th>
-                <th><strong>Read-only</strong></th>
-                <th><strong>Value-type</strong></th>
-                <th><strong>Represen-tation</strong></th>
-                <th><strong>Range</strong></th>
-                <th><strong>Description</strong></th>
-              </tr>
-            </thead>
-            <tbody>
-              <tr>
-                <td><em>deprecated</em> dcterms:type</td>
-                <td>zero-or-more</td>
-                <td>unspecified</td>
-                <td>String</td>
-                <td>n/a</td>
-                <td>n/a</td>
-                <td>A short string representation for the type, example <code>Defect</code>.</td>
-              </tr>
-              <tr>
-                <td>
-                  <strong>Relationship properties:</strong> This grouping of properties are used to identify
-                  relationships between resources managed by other OSLC servers
-                </td>
-                <td></td>
-                <td></td>
-                <td></td>
-                <td></td>
-                <td></td>
-                <td></td>
-              </tr>
-              <tr>
-                <td><em>deprecated</em> oslc_cm:testedByTestCase</td>
-                <td>zero-or-many</td>
-                <td>False</td>
-                <td>Resource</td>
-                <td>Reference</td>
-                <td><code style="white-space: pre;">any</code></td>
-                <td>
-                  Test case by which this change request is tested. It is likely that the target resource will be an
```

Comment:
I did not check each of the new entries here. But I assume this removed text is what is now available in the new text in the "specs/cm/change-mgt-shapes.ttl" file added above.